src/builtin/functions/color/opacity.rs

```diff
@@ -5,16 +5,84 @@ use crate::{
     value::Value,
 };
 
+fn is_ms_filter(s: &str) -> bool {
+    let mut chars = s.chars();
+
+    if let Some(c) = chars.next() {
+        if !matches!(c, 'a'..='z' | 'A'..='Z') {
+            return false;
+        }
+    } else {
+        return false;
+    }
```

Comment:
```suggestion
    if !matches!(chars.next(), Some('a'..='z') | Some('A'..='Z')) {
        return false;
    }
```

Reply:
Ah thank you. It seems I forgot that you can do ranges without `or patterns`

Reply:
For some reason this suggestion is marked as outdated, so I am unable to automatically commit it.

![image](https://user-images.githubusercontent.com/39542938/88941553-27351780-d257-11ea-8f67-91e0906c9f14.png)

Reply:
Okay, let me send some patch for this. I also found the outdated weird.

Reply:
https://github.com/connorskees/grass/pull/30